sessions/LIGHTING_TALKS.md

```diff
@@ -0,0 +1,58 @@
+## Session(s): Project Update Lightning Talks
+
+Short project updates in rapid succession. No more than 30-45 minutes at a time (6-9 updates). This can/should be done more than once at the meetings to give at least 12-18 projects exposure.
+
+**Proposed by** @flyingzumwalt
+**Led by** @flyingzumwalt
+**Duration** 30-45 minutes
+**Who Should Attend:** Everyone
+**Who has to be there in order for it to work:** People who want to give a project update.
+
+## Overview
+
+- **Objective**: Allow many projects to disseminate information across the entire set of attendees.
+- **Activity**: In a lecture format, allow a group of people to give pre-prepared 5-minute updates on their projects.
+- **Outcome**: Recordings of all the project updates and shared context for conference attendees to operate from.
+
+## Details
+
+This is a running list. Suggest more via PRs, or contact @flyingzumwalt to add something
+
+Here's a sketch of updates we could have on Day 1 and Day 2. People might want to do another session of updates during the unconference on Day 3.
+
+### Day 1, Monday
+
+| Topic/Project           | Presenter                   |
+|:-----------------------:|:---------------------------:|
+| IPFS ecosystem overview | @flyingzumwalt              |
+| js-ipfs                 | @diasdavid & @alanshaw      |
+| go-ipfs                 | TBD                         |
+| libp2p                  | @mgoelzer or @whyrusleeping |
+| IPLD                    | @stebalien                  |
+| peer🌟                  | @satazor (in lieu of @pgte) |
+| ipfs-cluster            | @hsanjuan                   |
+| infrastructure          | @lgierth or @eefahy         |
+| Web Browsers WG         | @lidel                      |
+
+
+### Day 2, Tuesday
+
+| Topic/Project           | Presenter                   |
+|:-----------------------:|:---------------------------:|
+| Data Stewardship & Data Together | @flyingzumwalt and @meiqimichelle |
+| textile.io ([Adding the world's personal data to IPFS](https://github.com/ipfs/developer-meetings/pull/2)) | @anderewxhill |
+| orbit-db                | @haadcode                   |
+| GUI WG                  | @olizilla                   |
+| DDC WG                  | @satazor & gpestana (in lieu @pgte) |
```

Comment:
@satazor, @gpestana put you on day two :)

Reply:
@gpestana can we switch the DDC WG for the first day? It seems more logical to present it first and have a follow up on peer-star afterwards

Reply:
@satazor yes, sure! It makes sense to me.